tests/utils/scanner_state_reporter_test.py

```diff
@@ -0,0 +1,40 @@
+"""Testing module for the scanner state reporter class."""
+
+import pytest
+
+from ostorlab.utils import scanner_state_reporter
+from ostorlab.utils.defintions import ScannerState
+
+
+class Memory:
+    percent: float = 10
+    total: int = 100
+
+
+@pytest.mark.asyncio
+async def testReportMethod_whenCalled_updateValuesCorrectly(mocker) -> None:
+    mocker.patch(
+        "ostorlab.apis.runners.authenticated_runner.AuthenticatedAPIRunner.execute",
+        eturn_value="dummy",
+    )
+    mocker.patch("psutil.cpu_percent", return_value=10)
+    mocker.patch("psutil.cpu_count", return_value=10)
+    mocker.patch("psutil.virtual_memory", return_value=Memory())
+    mock = mocker.patch("ostorlab.apis.add_scanner_state.AddScannerStateAPIRequest")
+    report = scanner_state_reporter.ScannerStateReporter(
+        scanner_id=1, scan_id=1, hostname="", ip="", errors=""
+    )
+    await report.report()
+    state = ScannerState(
```

Comment:
Can you please add new lines splitting the unit test into 3 parts: setup, call, asserts?